maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/LiferayMavenProjectProvider.java

```diff
@@ -128,10 +128,8 @@ public LiferayMavenProjectProvider()
         super( new Class<?>[] { IProject.class } );
     }
 
-    public IStatus doCreateNewProject( final NewLiferayPluginProjectOp op, IProgressMonitor monitor ) throws CoreException
+    public List<IProject> doCreateNewProject( final NewLiferayPluginProjectOp op, IProgressMonitor monitor ) throws CoreException
```

Comment:
Lets leave the return value a status object.  But instead lets have the calling method ccreate a new List<IProject> list and pass it into this method and the new projects created can be added to this list.